<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutWorldSkills</class>
 <widget class="QMainWindow" name="AboutWorldSkills">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="WorldSkillsLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>O WorldSkills</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HistoryButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>История</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CompetitionButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Компетенции</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ChempionshipsButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Чемпионаты</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="AboutFrame">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="AboutWSTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>741</width>
       <height>491</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Первый национальный чемпионат:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Весной 2013 года в Тольятти состоялся Первый всероссийский конкурс профессионального мастерства «Национальный чемпионат WorldSkills Russia — 2013». В чемпионате приняли участие более трёхсот участников в возрасте от 18 до 22 лет — студентов учреждений среднего профессионального образования, победителей региональных конкурсов.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;По результатам Национального чемпионата была сформирована сборная Российской Федерации, которая в июле 2013 года приняла участие в чемпионате мира WorldSkills International в Лейпциге (Германия).&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Основатель:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Albert Vidal&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;quot;Fill youth with enthusiasm through special action! Convince young people's parents, trainers and company chiefs that a promising future is possible only through good vocational training&amp;quot;.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- Francisco Albert-Vidal, IVTO President 1984-1992&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;It was his nephew, Juan Martinez Albert, and his friend, Jose Rubio, who asked for this nomination two years ago. &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Francisco Albert-Vidal was born in Pinoso in 1917 and worked in the city hall of the village of Petrer. Later, he was the Vocational Training General Secretary until 1977 as well as the founder of the International VET Competitions, now WorldSkills.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;He died in Madrid, in 1993, and was buried in the village of Pinoso.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;On 31 March 2015, the plenary session of the city hall of Pinoso approved, unanimously, his appointment as honorary citizen of the village, as recognition of his work in the field of vocational training for young people through the creation of important events like WorldSkills Competitions.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Words from Pepe Rubio (partner and friend): &amp;quot;Thanks for this recognition. Francisco Albert Vidal worked for universal values, for people and their education&amp;quot; &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Words from Juan Martinez (nephew): &amp;quot;I'm very excited, and I remember many family moments with him, especially together with my mother, the only sister of Francisco who is still alive. I want to transmit gratitude from my whole family.”&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Президент:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Simon Bartley&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(United Kingdom)&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Simon was elected President of WorldSkills International at the organization’s General Assembly in October 2010 and took up his four-year post on 10th October 2011, the day after the Closing Ceremony at WorldSkills London 2011. Previously Simon had been Chief Executive of UK Skills and WorldSkills London 2011 as well as being the UK’s Official Delegate to both WorldSkills International and WorldSkills Europe (on both of which he was also a Board Member). Prior to his involvement in UK Skills Simon worked in the Building Services Sector as Chairman of his family business and he now works as an international skills consultant. Simon was educated at Durham University, from where he graduated with a BSc in Engineering Science and Management and an MSc in Management Science. He is a Chartered Engineer, a Member of the Institution of Civil Engineers and a Fellow of City and Guilds. A Member of the UK Governments Skills Commission he is also on the Council of City &amp;amp; Guilds. Simon is a past Chair of the Confederation of British Industry’s Small and Medium Sized Enterprise Council.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ABackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="CompetitionFrame">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="CompetitionList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>256</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="AboutTextEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>401</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CompetitionLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Компетенции</string>
     </property>
    </widget>
    <widget class="QLabel" name="AboutLabel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать
 описание</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CoBackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ChempionshipsFrame">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ChampionshipsWSILabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Чемпионаты WSI</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ChampionshipsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>761</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ChBackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <zorder>ChempionshipsFrame</zorder>
   <zorder>MainFrame</zorder>
   <zorder>AboutFrame</zorder>
   <zorder>CompetitionFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
